scripts/src/tests/services/users.spec.ts

```diff
@@ -112,27 +112,27 @@ describe("api tests for v2 users", async () => {
 
 		// different appId
 		const user3 = await helpers.createUser({ appId: generateId(IdPrefix.App) });
-		await mock(app)
-			.get(`/v2/users/${ user3.appUserId }`)
+		await mock(expressApp)
+			.get(`/v2/users/${user3.appUserId}`)
 			.set("x-request-id", "123")
-			.set("Authorization", `Bearer ${ token.id }`)
+			.set("Authorization", `Bearer ${token.id}`)
 			.expect(404);
 	});
 
 	test("updateUser", async () => {
 		const testApp = await helpers.createApp(generateId(IdPrefix.App));
-		const newWalletAddress = "new_address_must_be_56_characters____bla___bla___bla____";
+		const newWalletAddress = `test_wallet_123456789012345678901234${ generateId() }`;
```

Comment:
i think that this misses the point.
your change makes the wallet address to have less than 56 chars.

and also, why changing it to begin with?

Reply:
It would not pass the new check this PR brings in.

Reply:
and no, it's not less than 56 chars hence why the tests pass

Reply:
well, you count on `generateId` to return a specific amount of chars, which isn't wise.
we might change that someday, and that will mess things up here.

Reply:
mmm I agree, but since there are other places in the tests I've seen it used in a similar manner (at least one instance is above) I don't think it makes any difference now. Also, I think if we do change the length it might make us check production code if our tests break. Although a specific test for this would be better.